ukrainian/make-a-hole/in-the-wall.md

```diff
@@ -1 +1,4 @@
-Оговтуюсь від шоку, беру виделку і виковирюю дирку у зефіровій стіні.. Що буде зі мною далі?
\ No newline at end of file
+... оговтуючись від шоку, виковирюю діркур у зефіровій стіні.. Що буде зі мною далі?
+
+[Загляну у дірку, мене завжди цікавило невідоме] (light/light.md)
+[В руці починає пекти] (fork/fork.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.